lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -2040,9 +2039,9 @@ def set_payload(resource_type, record, entry, is_json)
                              end
       elsif text_payload
         entry.text_payload = if @use_grpc
-                               convert_to_utf8(text_payload)
+                               convert_to_utf8(text_payload.to_s)
```

Comment:
Yeah, I was wondering what would happen if someone sent a `message` field that was not a String...

Would it make sense to change it above, e.g.:
```ruby
      elsif text_payload
        text_payload = text_payload.to_s
        entry.text_payload = if @use_grpc
                               convert_to_utf8(text_payload)
                             else
                               text_payload
                             end
```
?

Reply:
Done